### PR TITLE
restaurant and menu_item models set up along with tests and schema

### PR DESCRIPTION
**Pull request by eport01**
Database and relationships created. Restaurant and MenuItem models created along with basic tests. All tests pass

### CODE REVIEW

File: db/migrate/20230204044855_create_menu_items.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class CreateMenuItems < ActiveRecord::Migration[5.2]
+  def change
+    create_table :menu_items do |t|
+      t.belongs_to :restaurant, foreign_key: true
+      t.string :name
+      t.string :description
+      t.string :allergens
```
Comment (Yuji3000) [inline db/migrate/20230204044855_create_menu_items.rb]: Allergens should be changed to "tags"

File: db/migrate/20230204044855_create_menu_items.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      t.string :category
+      t.string :image
+      t.string :price
```
Comment (Yuji3000) [inline db/migrate/20230204044855_create_menu_items.rb]: Price should be a float rather than a string

File: app/models/menu_item.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class MenuItem < ApplicationRecord
```
Comment (Yuji3000) [inline app/models/menu_item.rb:1]: validations for our models?

File: db/migrate/20230204044855_create_menu_items.rb (hunk 1 above)
Comment (eport01) [reply db/migrate/20230204044855_create_menu_items.rb]: Updated!

File: db/migrate/20230204044855_create_menu_items.rb (hunk 2 above)
Comment (eport01) [reply db/migrate/20230204044855_create_menu_items.rb]: Also updated, thanks!

File: app/models/menu_item.rb (hunk 3 above)
Comment (eport01) [reply app/models/menu_item.rb:1]: Added validations and validation tests for our models!

**Comment by eport01** [comment]
Updated validations and tests for models, database to reflect tags instead of allergens, changed price to a float, and reseeded the database to reflect these changes. I also updated the server to localhost:3001 to reflect the json contract.

**Review by Yuji3000 — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**